src/pool/srv_pool.c

```diff
@@ -6978,7 +6976,7 @@ pool_svc_update_map_internal(struct pool_svc *svc, unsigned int opc,
 	/* For SWIM exclude, don't change pool map if the pw_rf is broken or is going to be broken,
 	 * with CRIT log message to ask administrator to bring back the engine.
 	 */
-	if (src == MUS_SWIM && opc == MAP_EXCLUDE) {
+	if (opc == MAP_EXCLUDE && (src == MUS_SWIM || !skip_rf_check)) {
```

Comment:
[Question] IIRC, `src` was introduced for the `dmg pool exclude` case. Could we remove `src`, `MUS_DMG`, and `MUS_SWIM` now that we have `skip_rf_check`?

Reply:
Indeed, I'll try to remove src.